blueprint/index.md

```diff
@@ -115,7 +116,7 @@ After the `terraform apply --auto-approve` command completes, you should see the
    ![Interaction details script](images/interaction-details-script.png "Interaction details script")
 2. Schedule a callback and end the interaction.
    ![Add callback](images/add-callback.png "Add callback")
-3. Callback should be added in **Admin > Performance > Scheduled Callbacks**.
+3. Add the callback in **Admin > Performance > Scheduled Callbacks**.
    ![Scheduled callback](images/scheduled-callback.png "Scheduled callback")
 4. Answer the callback and click **Begin Call**.
    ![Begin callback](images/begin-callback.png "Begin callback")
```

Comment:
After line 128, please add a link to this blueprint in the Github repository. See the bottom of this blueprint for an example: https://developer.genesys.cloud/blueprints/analytics-detail-record-metrics-blueprint/